tests/test_coriolis.py

```diff
@@ -0,0 +1,56 @@
+import numpy as np
+import pytest
+from pytest import param as p
+
+from jaxsim.high_level.common import VelRepr
+from jaxsim.high_level.model import Model
+
+from . import utils_models, utils_rng
+from .utils_models import Robot
+
+
+@pytest.mark.parametrize(
+    "robot, vel_repr",
+    [
+        p(*[Robot.DoublePendulum, VelRepr.Inertial], id="DoublePendulum-Inertial"),
+        p(*[Robot.DoublePendulum, VelRepr.Body], id="DoublePendulum-Body"),
+        p(*[Robot.DoublePendulum, VelRepr.Mixed], id="DoublePendulum-Mixed"),
+        p(*[Robot.Ur10, VelRepr.Inertial], id="Ur10-Inertial"),
+        p(*[Robot.Ur10, VelRepr.Body], id="Ur10-Body"),
+        p(*[Robot.Ur10, VelRepr.Mixed], id="Ur10-Mixed"),
+        p(*[Robot.AnymalC, VelRepr.Inertial], id="AnymalC-Inertial"),
+        p(*[Robot.AnymalC, VelRepr.Body], id="AnymalC-Body"),
+        p(*[Robot.AnymalC, VelRepr.Mixed], id="AnymalC-Mixed"),
+        p(*[Robot.Cassie, VelRepr.Inertial], id="Cassie-Inertial"),
+        p(*[Robot.Cassie, VelRepr.Body], id="Cassie-Body"),
+        p(*[Robot.Cassie, VelRepr.Mixed], id="Cassie-Mixed"),
+    ],
+)
+def test_coriolis(robot: utils_models.Robot, vel_repr: VelRepr) -> None:
+    """
+    Unit test of the Coriolis matrix.
+    """
+
+    # Initialize the gravity
+    gravity = np.array([0, 0, -10.0])
+
+    # Get the URDF of the robot
+    urdf_file_path = utils_models.ModelFactory.get_model_description(robot=robot)
+
+    # Build the high-level model
+    model = Model.build_from_model_description(
+        model_description=urdf_file_path,
+        vel_repr=vel_repr,
+        gravity=gravity,
+        is_urdf=True,
+    ).mutable(mutable=True, validate=True)
+
+    # Initialize the model with a random state
+    model.data.model_state = utils_rng.random_physics_model_state(
+        physics_model=model.physics_model
+    )
+
+    # Compute model acceleration with ABA
+    H, H_dot, C = model.coriolis_matrix()
```

Comment:
```suggestion
    H, H_dot, C = model.coriolis_matrix()
```

Could it make sense to add a test here to check that the mass matrix here is the same that the mass matrix computed with CRBA?

Reply:
Good idea @traversaro.

@flferretti please note that the low-level [`jaxsim.physics.algos.crba.crba`](https://github.com/ami-iit/jaxsim/blob/ed05df47a0484280d478f33712e34a69ccb67d4c/src/jaxsim/high_level/model.py#L829-L832) always computes $M_B(\mathbf{s})$ in body-fixed representation since it's the only representation that is independent from the base pose. Then, the rest of the `high_level` logic converts the body-fixed mass matrix to the active representation.

Make sure to compare the same quantities.